pvacseq/lib/main.py

```diff
@@ -128,6 +128,12 @@ def main(args_input = sys.argv[1:]):
              + "Needs to be an even number.",
     )
     parser.add_argument(
+        "-d", "--downstream-sequence-length",
+        default='1000',
+        help="Cap to limit the downstream sequence length for frameshifts when creating the fasta file. "
+            + "Use 'full' to include the full downstream sequence. Default: 1000"
+    )
```

Comment:
I felt like `full` might be a bit more user friendly than `-1` but it does make for some oddity in the handling of this parameter. @chrisamiller any preference on how to encode that the user wants to use the full downstream sequence?